tools/build_tar/buildtar_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "testing"
+
+func TestExtractDebPackageName(t *testing.T) {
+	cases := []struct{
+		metadata string
+		filename string
+		expectedPackageName string
+	}{
+		{
+			"Package: test\nDescription: Dummy\nVersion: 1.2.4",
+			"test.deb",
```

Comment:
maybe make this `test-valid.deb` just so we're certain we're not using the fallback case?